CMakeLists.txt

```diff
@@ -4,7 +4,6 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/CMake)
 find_package(CUnit REQUIRED)
 find_package(FreeImage REQUIRED)
 find_package(GLEW REQUIRED)
-find_package(IniParser REQUIRED)
```

Comment:
Yes, add it as a required package. Better to get the error when generating the project then halfway through the build.